guides/common/modules/proc_configuring-an-alternative-directory-to-execute-remote-jobs-on-clients.adoc

```diff
@@ -2,8 +2,18 @@
 
 = Configuring {Project} to Use an Alternative Directory to Execute Remote Jobs on Hosts
 
+ifeval::["{context}" == "managing-hosts"]
+
 By default, {Project} uses the `/var/tmp` directory on the client system to execute the remote execution jobs. If the client system has `noexec` set for the `/var/` volume or file system, you must configure {Project} to use an alternative directory because otherwise the remote execution job fails since the script cannot be run.
 
+endif::[]
+
+ifeval::["{context}" == "ansible"]
+
+By default, {Project} uses the `/var/tmp` directory on the client system to execute the remote execution jobs. Ansible puts its own files it requires into the `$HOME/.ansible/tmp` directory, where `$HOME` is the home directory of the remote user.
```

Comment:
For ansible, nothing should be going into `/var/tmp` so we can drop the entire sentence